main/pipeline/pipeline.cc

```diff
@@ -350,8 +350,7 @@ const unique_ptr<parser::Node> convertPrismToSorbet(pm_node_t *node, pm_parser_t
     }
 }
 
-unique_ptr<parser::Node> runPrismParser(core::GlobalState &gs, core::FileRef file, const options::Printers &print,
-                                        bool traceLexer, bool traceParser) {
+unique_ptr<parser::Node> runPrismParser(core::GlobalState &gs, core::FileRef file) {
```

Comment:
Simplified the method signature to remove things I'm not using (yet) for easy use in tests.